src/pages/CubeAnalysisPage.js

```diff
@@ -126,6 +154,10 @@ const CubeAnalysisPage = ({
         />
       ),
     },
+    {
+      name: 'Playtest',
```

Comment:
I do not like this name. It is a verb and means something different from the other places we use it.